vertical-pod-autoscaler/examples/hamster-default-recommender.yaml

```diff
@@ -0,0 +1,56 @@
+# This config creates a deployment with two pods, each requesting 100 millicores
+# and trying to utilize slightly above 500 millicores (repeatedly using CPU for
+# 0.5s and sleeping 0.5s).
+# It also creates a corresponding Vertical Pod Autoscaler that adjusts the
+# requests.
+# Note that the update mode is left unset, so it defaults to "Auto" mode.
+---
+apiVersion: "autoscaling.k8s.io/v1"
+kind: VerticalPodAutoscaler
+metadata:
+  name: hamster-default-recommender-vpa
+spec:
+  recommenderName: ['default']
```

Comment:
Why do we need this example?

If this is to show that the default recommender might be specified explicitly, I think it would be good to state here (and maybe in the hamster.yaml) that recommenderName does not have to be set.
But maybe it would be enough to simply modify hamster.yaml with the comment how to specify the recommender.

Reply:
@kgolab , yes, I will just add comments in the default `hamster.yaml`